<a href="https://colab.research.google.com/github/Hyeytt/basic_python221028/blob/main/basic_python221028.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [11]:
x = int(input("x의 값은? : "))
y = int(input("y의 값은? : "))

print("두수의 합은 : ", x+y )         # 구분자 : 쉼표
print("두수의 차는 : ", x-y )
print("두수의 곱은 : ", x*y)
print("두수의 평균은 :" ,(x+y)/2)
print("큰 수는? : " , max(x, y))
print("작은 수는?: " , min(x,y))

x의 값은? : 10
y의 값은? : 5
두수의 합은 :  15
두수의 차는 :  5
두수의 곱은 :  50
두수의 평균은 : 7.5
큰 수는? :  10
작은 수는?:  5


In [4]:
r = float(input("r의 값은? : "))
h = float(input("h의 값은? : "))

vol = 3.141592 * r**2 * h
print("원기둥의 부피는" , vol )

r의 값은? : 10
h의 값은? : 10
원기둥의 부피는 3141.592


In [7]:
number  = int(input("정수를 입력하시오 : "))

sum = 0     # 초기화 시키기 , 처음에 시작할때 
sum = sum + number % 10
number = number // 10 

sum = sum + number % 10
number = number // 10

sum = sum + number % 10 
number = number // 10 

sum = sum + number % 10
number = number // 10 

print(" 자리수의 합은 : ", str(sum))

정수를 입력하시오 : 1234
 자리수의 합은 :  10


In [10]:
x1 = int(input("x1의 값을 입력하시오 : "))
y1 = int(input("y1의 값을 입력하시오 : "))
x2 = int(input("x2의 값을 입력하시오 : "))
y2 = int(input("y2의 값을 입력하시오 : "))
dist = ((x2 - x1)**2 +(y2 - y1)**2)**0.5
print("두 점 사이의 거리는", dist , "입니다")


x1의 값을 입력하시오 : 0
y1의 값을 입력하시오 : 0
x2의 값을 입력하시오 : 100
y2의 값을 입력하시오 : 100
두 점 사이의 거리는 141.4213562373095 입니다


import turtle

 t = turtle.Turtle()


 t.shape("turtle")
 
t.left(45)

 t.fd(141)

 t.setheading(0)

 t.goto(0,0)

 t.fd(100)

 t.lt(90)

 t.fd(100)


    import turtle

    t = turtle.Turtle()

    t.shape("turtle")

    t.left(45)

    t.fd(141)

    t.setheading(0)

    t.goto(0,0)

    t.fd(100)

    t.lt(90)

    t.fd(100)

    t. _screen.exitonclick()

import turtle

t = turtle.Turtle()


t.shape("turtle")

x1 = int(input("x1 : "))


y1 = int(input("y1 : "))


x2 = int(input("x2 : "))


y2 = int(input("y2 : "))



dist =((x1 - x2)** 2  + (y1 - y2)**2 )**0.5


t.up()


t.goto(x1,y1)


t.down

t.goto(x2,y2)

t.write("점의 길이 = ", str(dist))
